src/schema/objects/metadata.yaml

```diff
@@ -2647,6 +2718,7 @@ _LandmarkCoordinates:
     maxItems: 3
 _MEGCoordSys:
   type: string
+  # TODO: Add definitions for these values.
```

Comment:
What do the different `_MEGCoordSys` values mean?

Reply:
see https://github.com/bids-standard/bids-specification/pull/919/files#r785334647